package.json

```diff
@@ -5,6 +5,7 @@
   "homepage": "https://perseids-publications.github.io/treebank-template/",
   "dependencies": {
     "@primer/octicons-react": "^9.4.0",
+    "alpheios-messaging": "https://github.com/alpheios-project/alpheios-messaging",
```

Comment:
Do you know if this will eventually be published in an npm package?

Reply:
eventually yes, I think we aren't quite ready to yet but soon.